erpnext/selling/doctype/sales_order/sales_order.json

```diff
@@ -1240,12 +1248,18 @@
    "fieldname": "signature",
    "fieldtype": "Signature",
    "label": "Signature"
+  },
+  {
+   "default": "0",
+   "fieldname": "ignore_pick_list",
```

Comment:
why did you add this ?

Reply:
To ignore pick list status if checked then it set statuses the old way.